doc/tutorials/gapi/face_beautification/face_beautification.markdown

```diff
@@ -0,0 +1,500 @@
+# Implementing a face beautification algorithm on G-API {#tutorial_gapi_face_beautification}
+
+@prev_tutorial{tutorial_gapi_anisotropic_segmentation}
+
+[TOC]
+
+# Introduction {#gapi_fb_intro}
+
+In this tutorial you are going to learn:
+* How to create a custom kernel implementation (using OpenCV backend);
```

Comment:
I suggest `How to implement a custom kernel`

Reply:
OK